### PR TITLE
Bug 1796535: Handle SDN CRDs as regular crd types without special rules

### PR DESCRIPTION
**Pull request by abhat**
Previously, it was believed that the sdn crds need special exception
handling. But it turns out that the crds were missing the preserveUnknownFields
flag. Adding that flag back obviates the need for special handling.

### CODE REVIEW

File: test/extended/cli/explain.go [hunk 1]
```diff
@@ -375,11 +359,11 @@ var _ = g.Describe("[cli] oc explain networking types", func() {
 
 	oc := exutil.NewCLI("oc-explain", exutil.KubeConfigPath())
 
-	g.It("should contain proper fields description for special networking types", func() {
-		for _, st := range specialNetworkingTypes {
-			e2e.Logf("Checking %s, Field=%s...", st.gv, st.field)
-			o.Expect(verifyExplain(oc, nil, schema.GroupVersionResource{},
-				st.pattern, st.field, fmt.Sprintf("--api-version=%s", st.gv))).NotTo(o.HaveOccurred())
+	g.It("should contain proper spec+status for networking CRDs", func() {
```
Comment (soltysh) [inline test/extended/cli/explain.go]: If networking is now following the usual types, I'd expect it to be part of `crdTypes` as the others are. No exceptions.

Comment (abhat) [reply test/extended/cli/explain.go]: Yes, but we need to split the test out since these crds are applicable only for Openshift SDN and not OVN.

Comment (soltysh) [reply test/extended/cli/explain.go]: :+1:

**Comment by abhat** [comment]
@soltysh can you approve please?

**Comment by abhat** [comment]
Upon closer examination, some of the crds have spec, but no status. It's best to leave them with special handling. Closing the PR.

**Pull request closed without merging**